packages/stats/gbstats/bayesian/tests.py

```diff
@@ -223,3 +227,114 @@ def get_risk(mu, sigma) -> List[float]:
         risk_ctrl = float((1.0 - prob_ctrl_is_better) * mn_pos)
         risk_trt = -float(prob_ctrl_is_better * mn_neg)
         return [risk_ctrl, risk_trt]
+
+
+@dataclass
+class BanditConfig(BayesianConfig):
+    burn: int = 0
+    iteration: int = 0
+    top_two: bool = True
+    prior_distribution: GaussianPrior = field(default_factory=GaussianPrior)
+
+
+class Bandits(object):
+    def __init__(
+        self,
+        stat: Sequence[Union[ProportionStatistic, SampleMeanStatistic]],
+        config: BanditConfig,
+    ):
+        self.stat = stat
+        self.config = config
+        self.n_variations = len(stat)
+
+    @property
+    def variation_means(self):
```

Comment:
For these and other methods, is a return type hint feasible? I know numpy can be annoying with typing, but lmk if you can add type hinting.

Reply:
done